src/go/mongolib/util/util.go

```diff
@@ -134,7 +140,7 @@ func GetHostnames(ctx context.Context, client *mongo.Client) ([]string, error) {
 		}
 	}
 
-	return nil, fmt.Errorf("cannot get shards map")
+	return nil, nil // standalone instance
```

Comment:
when it's possible to not get shardingNotEnabledError on standalone instance?
Is it possible that sharding is enabled, but we got empty `shardsMap`?

Reply:
@percona-csalguero ^

Reply:
AFAIK, no

Reply:
So why do we have these conditions above?